opm/grid/cpgrid/CpGridData.cpp

```diff
@@ -878,6 +814,64 @@ void CpGridData::distributeGlobalGrid(const CpGrid& grid,
 
     logical_cartesian_size_=view_data.logical_cartesian_size_;
 
+        // Set up the new topology arrays
+    // Count the existing points and allocate space
+    EntityVariable<cpgrid::Geometry<0, 3>, 3>& point_geom = geometry_.geomVector(std::integral_constant<int,3>());
+    const std::vector<cpgrid::Geometry<0, 3> >& global_point_geom=view_data.geomVector<3>();
+    point_geom.reserve(noExistingPoints);
+
+    // Now copy the point geometries that do exist.
+    for(auto begin=point_indicator.begin(), pi=begin,  end=point_indicator.end(); pi!=end;
+        ++pi)
+    {
+        if(*pi<std::numeric_limits<int>::max())
+        {
+            point_geom.emplace_back(global_point_geom[pi-begin]);
+        }
+    }
+
```

Comment:
Nitpick: extra blank

Reply:
I have no idea what you mean by that. Extra line?

Reply:
Finally got it. Blank before and after the minus sign. Amended last commit

Reply:
Sorry for being too brief, I actually meant extra blank *line*.